templates/meta/patient_password_reset.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "patient_password_reset",
+    "description": "email for patients that requested password reset",
+    "templateFilename": "patient_password_reset.html",
+    "subject": "PatientPasswordResetSubject",
+    "contentParts":[
+        "PatientPasswordResetHello",
+        "PatientPasswordResetRequest",
+        "PatientPasswordResetDidNotRequest",
+        "PatientPasswordResetByDevice",
+        "PatientPasswordResetClick",
+        "FooterOpenSource",
+        "FooterGetSupport",
+        "FooterSincerely",
+        "FooterSignature",
+        "FooterMadePossible"
```

Comment:
```suggestion

```

Reply:
because not needed in DBL Look&Feel that has been merged into master in the meantime